micropython/lora/examples/reliable_delivery/sender.py

```diff
@@ -149,7 +149,7 @@ def send(self, sensor_data, adjust_output_power=True):
                 delta = time.ticks_diff(maybe_ack.ticks_ms, sent_at)
                 print(
                     f"ACKed with RSSI {rssi}, {delta}ms after sent "
-                    + f"(skew {delta-ACK_DELAY_MS-ack_packet_ms}ms)"
+                    f"(skew {delta-ACK_DELAY_MS-ack_packet_ms}ms)"
```

Comment:
MicroPython doesn't support adjacent f-strings, it needs the "+".

(This file is not tested by the CI here, because it's an example.)